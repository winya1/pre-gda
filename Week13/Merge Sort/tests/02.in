4
4
69 99 31 68
5
56 77 31 92 19
10
65 57 85 47 95 69 49 28 60 97
100
22 42 4 37 89 53 10 69 75 36 85 81 40 87 34 40 24 82 65 71 80 60 14 86 7 27 96 63 74 61 2 54 86 91 10 59 95 4 61 69 10 33 26 48 54 78 40 72 50 65 42 74 18 51 3 41 16 24 16 71 58 56 51 1 97 51 96 85 78 90 12 2 4 14 85 70 15 16 63 4 98 87 9 23 8 63 24 18 31 65 59 10 63 28 40 95 37 80 13 77
