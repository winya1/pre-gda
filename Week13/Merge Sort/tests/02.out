Case #1: [31, 68, 69, 99]
Case #2: [19, 31, 56, 77, 92]
Case #3: [28, 47, 49, 57, 60, 65, 69, 85, 95, 97]
Case #4: [1, 2, 2, 3, 4, 4, 4, 4, 7, 8, 9, 10, 10, 10, 10, 12, 13, 14, 14, 15, 16, 16, 16, 18, 18, 22, 23, 24, 24, 24, 26, 27, 28, 31, 33, 34, 36, 37, 37, 40, 40, 40, 40, 41, 42, 42, 48, 50, 51, 51, 51, 53, 54, 54, 56, 58, 59, 59, 60, 61, 61, 63, 63, 63, 63, 65, 65, 65, 69, 69, 70, 71, 71, 72, 74, 74, 75, 77, 78, 78, 80, 80, 81, 82, 85, 85, 85, 86, 86, 87, 87, 89, 90, 91, 95, 95, 96, 96, 97, 98]
